src/main/java/lotto/domain/LottoWinningType.java

```diff
@@ -0,0 +1,51 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum LottoWinningType {
+
+    FIRST(6, 2000000000),
```

Comment:
언더스코어(`_`)를 활용하면 단위가 큰 숫자의 가독성을 높여볼 수 있을 것 같아요.
```suggestion
    FIRST(6, 2_000_000_000),
```

Reply:
좋은 제안 감사드립니다 👍🏼